src/sinol_make/commands/run/__init__.py

```diff
@@ -127,15 +128,20 @@ def print_table_end():
                     if status == "  ":
                         group_status = "  "
                         min_points = 0
+                        all_tests_finished = False
                     else:
                         group_status = update_group_status(group_status, status)
 
-                points = math.floor(min_points / 100 * scores[group])
-                print("%3s" % util.bold(util.color_green(group_status)) if group_status == "OK" else util.bold(
-                    util.color_red(group_status)),
-                      "%3s/%3s" % (points, scores[group]),
-                      end=" | ")
-                program_scores[program] += scores[group] if group_status == "OK" else 0
+                points = math.ceil(min_points / 100 * scores[group])
+                if not all_tests_finished:
```

Comment:
```suggestion
                all_tests_finished = all([results[test].Status != "  " for test in results])
                if not all_tests_finished:
```
The "create a bool, update in in a for, use it further down" is a code pattern that reduces readability and increases chances of a bug (there are three places where a bug could happen, instead of one). Please try to reduce the amount of variables with wide scope in the code.

The suggestion was hand-written, it might not work.

The `status == "  "` is also not readable and seems like an easy place to make a bug. Is there any way to make it clearer?

Reply:
After improving the `status == "  "` case, the meaning of the `all(...)` statement would be so clear that a variable might not needed and just an `if not all(...)` would be perfectly readable and shorter.

Reply:
I changed it a bit